packages/create-llama/templates/components/engines/python/agent/__init__.py

```diff
@@ -44,7 +39,7 @@ def get_chat_engine():
     tools += ToolFactory.from_env()
 
     return create_agent_from_llm(
```

Comment:
Better use `AgentRunner.from_llm` now and remove the factory (everywhere)
```suggestion
    return AgentRunner.from_llm(
```

Reply:
Nice, so it's already there. Let me update this.